### PR TITLE
Added FAQs 

### PR DESCRIPTION
**Pull request by ank1064**
Added 50 FAQs

### CODE REVIEW

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 1]
```diff
@@ -810,3 +810,29 @@ To create an annual budget with a monthly breakdown, you need to select the budg
 
 Yes, since budgets are based on Perspectives, we can also incorporate built-in cloud discounts.
 
+#### Why are my connectors failing with error - Please install the metrics server on your cluster?
```
Comment (brandon-harness) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:813]: remove ?

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 2]
```diff
@@ -810,3 +810,29 @@ To create an annual budget with a monthly breakdown, you need to select the budg
...
+
+The Metrics Server must be running on the Kubernetes cluster where your Harness Kubernetes Delegate is installed. Before enabling CCM for Kubernetes, you must make sure the utilization data for pods and nodes is available.
```
Comment (brandon-harness) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:815]: replace 'is' with 'are'

File: kb/cloud-cost-management/cloud-cost-management-faqs.md [hunk 3]
```diff
@@ -810,3 +810,29 @@ To create an annual budget with a monthly breakdown, you need to select the budg
...
+
+#### What is the metrics server for CCM?
+
+The Metrics Server is a cluster-wide aggregator of resource usage data. It collects resource metrics from kubelets and exposes them in the Kubernetes API server through Metrics API. CCM polls the utilization data every minute on the Delegate. The metrics are aggregated for 20 minutes and then CCM keeps one data point per 20 minutes. For more information, see Installing the Kubernetes Metrics Server from AWS.
```
Comment (brandon-harness) [inline kb/cloud-cost-management/cloud-cost-management-faqs.md:819]: Add a link to the github repo or link 'Installing the Kubernetes Metrics Server from AWS'

File: kb/continuous-delivery/faqs.md [hunk 4]
```diff
@@ -3293,6 +3293,203 @@ Yes, the skip keywords can be placed anywhere within the commit message, includi
 #### Does harness pipeline support triggering GitHub action workflow from the pipeline?
 As per the current design, there's no native step for this but user can write a shell script at the end of execution to trigger the GitHub action workflow.
 
+#### How do I generate a manifest with secrets without marking it?
+
+As per the current design, The manifests secrets are masked as `***`. In the output, the secrets are replaced with the string `Kioq`.
```
Comment (brandon-harness) [inline kb/continuous-delivery/faqs.md:3298]: lowercase 'T'.  'the'

File: kb/continuous-delivery/faqs.md [hunk 5]
```diff
@@ -3293,6 +3293,203 @@ Yes, the skip keywords can be placed anywhere within the commit message, includi
...
+- key: <+eventPayload.after>
+operator: NotEquals
+value: "0000000000000000000000000000000000000000"
+headerConditions:
+- key: <+trigger.header['X-GitHub-Event']>
+operator: Equals
+value: Push
+```
+
+#### Does harness can ignore fake commits that have a SHA value - "0000....."
```
Comment (brandon-harness) [inline kb/continuous-delivery/faqs.md:3313]: This is a question.  Remove the word 'can' and add a question mark

**Comment by krishi0408** [comment]
@ank1064 please fix the build failure.

**Comment by dewan-ahmed** [comment]
A gentle reminder that if the build failures and not addressed on time, stalebot will close this PR after certain days of inactivity.

**Comment by bot2-harness** [comment]
This pull request has been automatically closed due to 30 days of inactivity. If you believe this was done in error, please feel free to reopen the PR or reach out to the BT team for further assistance. Thank you for your contributions!

**Pull request closed without merging**